docs/dev/framework/templates/twig.md

```diff
@@ -290,24 +291,41 @@ template.
 
 ⚠️ There is a small difference when it comes to callables like anonymous functions
 or closures. Every element in Twig's context needs to be a literal or an object,
-so we're wrapping functions into objects with `__toString()` functionality.
-In case you need to supply arguments or you need anything else than the string
-output, you'll need to add `.invoke()` to the variable name:
+so when overwriting a PHP Template, we're wrapping any function in the template
+data into an object with `__toString()` functionality. In case you need to
+supply arguments, or you need anything else than string output, you'll need to
+add `.invoke()` to the variable name:
+
+```php
+// Template data, that was set for a PHP template 
+$this->Template->setData([
+    'normalValue' => 'foo',
+    'lazyValue' => static function(): string {
+        return 'foo';
+    },
+    'fooFunction' => static function(string $value): string {
+        return "foo-$value";
+    },
+     'lazyArray' => static function(): array {
+        return [1, 2, 3, 4, 5];
+    },
+]);
+```
 
 {{< tabs groupId="twig">}}
 {{% tab name="PHP" %}}
 ```html
 <?php echo $this->normalValue; ?>
-<?php echo $this->lazyValue1(); ?>
-<?php echo $this->lazyValue2('foo'); ?>
+<?php echo $this->lazyValue(); ?>
+<?php echo $this->fooFunction('bar'); ?>
 <?php echo implode(', ', $this->lazyArray()); ?>
```

Comment:
```suggestion
<?php echo $this->lazyValue; ?>
<?php echo $this->fooFunction('bar'); ?>
<?php echo implode(', ', $this->lazyArray); ?>
```
You don’t need the braces in the legacy templates.

Reply:
Thanks, wasn't sure about this anymore!

fixed in 289627d